### PR TITLE
Governance: Min tokens to create governance

### PR DESCRIPTION
**Pull request by SebastianBor**
#### Summary

Replace realm Custodian role with `min_tokens_to_create_governance` to put it in hands of all community members and keep consistent with min_tokens_to_create_proposal

### CODE REVIEW

File: governance/program/src/instruction.rs [hunk 1]
```diff
@@ -50,7 +50,7 @@ pub enum GovernanceInstruction {
     /// 5. `[]` System
     /// 6. `[]` SPL Token
     /// 7. `[]` Sysvar Rent
-    /// 8. `[]` Realm custodian - optional    
+
```
Comment (joncinque) [inline governance/program/src/instruction.rs:53]: nit: numbering here?

File: governance/program/src/instruction.rs [hunk 2]
```diff
@@ -383,7 +387,7 @@ pub enum GovernanceInstruction {
     /// Sets realm config
     ///   0. `[writable]` Realm account
     ///   1. `[signer]`  Realm authority    
-    ///   2. `[]` Realm custodian - optional    
+
```
Comment (joncinque) [inline governance/program/src/instruction.rs:390]: nit: numbering

File: governance/program/src/instruction.rs [hunk 3]
```diff
@@ -405,10 +409,10 @@ pub fn create_realm(
     realm_authority: &Pubkey,
     community_token_mint: &Pubkey,
     payer: &Pubkey,
-    realm_custodian: Option<Pubkey>,
```
Comment (joncinque) [inline governance/program/src/instruction.rs:408]: So much for my comment on the last review -- no worries!

File: governance/program/src/processor/process_create_account_governance.rs [hunk 4]
```diff
@@ -29,10 +29,13 @@ pub fn process_create_account_governance(
     let realm_info = next_account_info(account_info_iter)?; // 0
     let account_governance_info = next_account_info(account_info_iter)?; // 1
     let governed_account_info = next_account_info(account_info_iter)?; // 2
-    let payer_info = next_account_info(account_info_iter)?; // 3
-    let system_info = next_account_info(account_info_iter)?; // 4
 
-    let rent_sysvar_info = next_account_info(account_info_iter)?; // 5
+    let _token_owner_record_info = next_account_info(account_info_iter)?; // 3
```
Comment (joncinque) [inline governance/program/src/processor/process_create_account_governance.rs:33]: What are all of these for? Is it worth adding any checks even at this point or is it ok for junk to be provided?

File: governance/program/src/state/realm.rs [hunk 5]
```diff
@@ -21,9 +21,8 @@ pub struct RealmConfigArgs {
     /// If yes then council_mint account must also be passed to the instruction
     pub use_council_mint: bool,
 
-    /// Indicates whether custodian should be used
-    /// If yes then custodian account must also be passed to the instruction  
-    pub use_custodian: bool,
```
Comment (joncinque) [inline governance/program/src/state/realm.rs:26]: nice, this seems much cleaner!

**Review by joncinque — COMMENTED** [review]
Is this is an in-between PR to be built on soon?

**Comment by SebastianBor** [comment]
> Is this is an in-between PR to be built on soon?

yes, it was an  in-between PR. I'm sorry for not being more clear about that.

File: governance/program/src/instruction.rs (hunk 1 above)
Comment (SebastianBor) [reply governance/program/src/instruction.rs:53]: fixed

File: governance/program/src/instruction.rs (hunk 2 above)
Comment (SebastianBor) [reply governance/program/src/instruction.rs:390]: fixed

**Pull request merged**